### ISSUE TITLE
Rollback causes all versions of MK2 (arcade) to close after opening

### ISSUE THREAD

**Issue by SonsofOcelot** [MisterTea/MAMEHub#341]
I have tested NBA jam, trog, SF2CE, and a few others and all worked fine. Mortal Kombat 2 th though is just closing 3 seconds after opening.

It runs fine with rollback set to 0. I've made no other changes to the INI file either. My system specs are Win7 ultimate x64, Intel core i5-4570, 8gb ddr3 ram 1600mhz.  So plenty of ram and power to support rollback.

I can supply logs if you want when I get home just tell me where to upload them.

**Comment by MisterTea** [comment]
Please put logs in pastebin and attach the link to this issue

**Comment by SonsofOcelot** [comment]
http://pastebin.com/q9V0thM1

MK2 log file ^

**Comment by MisterTea** [comment]
Wow!  That's a very interesting crash.  It doesn't happen without rollback enabled?

**Comment by Samus428** [comment]
I reported that issue a few days ago, no response til this one

**Comment by SonsofOcelot** [comment]
Nope, it runs absolutely fine with rollback set to 0. Prinygod also told me that it crashed on him, and my buddy Lonewulf89 has the exact same error. Think I'm gonna try MK1, MK3, and the European MK2 tonight and see if I get the same error

**Comment by Samus428** [comment]
mk1 and umk3 runs fine with rollback set to 1 , i only have the problem with mk2 in all its variations with rollback  set to 1. I just set rollback set to 0 and mk2 works now , but still have this awful input delay that had in the past few months

**Comment by SonsofOcelot** [comment]
Well in regards to your input lag I changed the "base delay" setting to -1 and that fixed that issue... But I still have yet to see if rollback interferes with anything else.

Also worth noting Valnar22 reported to me on client chat that MK2 acted the same way for him and he's still running x32  v.154 of UME. I'll see if I can get his log and post it to see if it's the same error

**Comment by Samus428** [comment]
Settting base delay" setting to -1 does not fix the input lag
